include/extractor/turn_instructions.hpp

```diff
@@ -9,21 +13,22 @@ namespace extractor
 enum class TurnInstruction : unsigned char
 {
     NoTurn = 0,
-    GoStraight,
-    TurnSlightRight,
-    TurnRight,
-    TurnSharpRight,
-    UTurn,
-    TurnSharpLeft,
-    TurnLeft,
-    TurnSlightLeft,
-    ReachViaLocation,
-    HeadOn,
+    GoStraight,       // 1
+    TurnSlightRight,  // 2
+    TurnRight,        // 3
+    TurnSharpRight,   // 4
+    UTurn,            // 5
+    TurnSharpLeft,    // 6
+    TurnLeft,         // 7
+    TurnSlightLeft,   // 8
+    ReachViaLocation, // 9
+    HeadOn,           // 10
     EnterRoundAbout,
     LeaveRoundAbout,
     StayOnRoundAbout,
     StartAtEndOfStreet,
     ReachedYourDestination,
+    NameChanges,
```

Comment:
This NameChanges instruction has to be handled in front-ends, if we decide to merge this, or we would have to fake it/deactivate it.